lib/src/util.dart

```diff
@@ -270,6 +272,36 @@ final _usesOverReactRegex = RegExp(
   multiLine: true,
 );
 
+VersionRange generateNewVersionRange(
+    VersionRange currentRange, VersionRange targetRange,
+    {bool shouldOverrideLowerBound = false}) {
+  if (shouldOverrideLowerBound) {
+    return targetRange;
+  } else {
+    String versionRange;
+
+    if (currentRange.min > targetRange.min) {
+      versionRange = '>=${currentRange.min.toString()}';
```

Comment:
#nit toString is unnecessary when string interpolating
```suggestion
      versionRange = '>=${currentRange.min}';
```